src/logic/small.lean

```diff
@@ -74,9 +74,16 @@ theorem small_congr {α : Type*} {β : Type*} (e : α ≃ β) : small.{w} α ↔
 instance small_subtype (α : Type v) [small.{w} α] (P : α → Prop) : small.{w} { x // P x } :=
 small_map (equiv_shrink α).subtype_equiv_of_subtype'
 
-theorem small_of_injective {α : Type*} {β : Type*} [small.{w} β]
-  (f : α → β) (hf : function.injective f) : small.{w} α :=
-small_map (equiv.of_injective f hf)
+theorem small_of_injective {α : Type v} {β : Type w} [small.{u} β]
+  (f : α → β) (hf : function.injective f) : small.{u} α :=
+begin
+  rw small_congr (equiv.of_injective f hf),
+  apply_instance,
+end
```

Comment:
I think that the proof of `small_of_injective` should be reverted.

Reply:
Otherwise LGTM.
bors d+

Reply:
One last question, actually. Should `small_range` be made into an `instance`? Or should it remain a theorem?

Reply:
Making it into an instance made it much more convenient when I tried it out on another branch. If this isn't the correct approach, I'll revert it in another PR.